lib/util.js

```diff
@@ -403,8 +403,10 @@ function formatValue(ctx, value, recurseTimes) {
   // Some type of object without properties can be shortcutted.
   if (keys.length === 0) {
     if (typeof value === 'function') {
-      return ctx.stylize(`[Function${value.name ? `: ${value.name}` : ''}]`,
-                         'special');
+      const asyncPrefix = binding.isAsyncFunction(value) ? 'async ' : '';
+      return ctx.stylize(
+        `[${asyncPrefix}Function${value.name ? `: ${value.name}` : ''}]`,
```

Comment:
I mean, if I change the code to use constructor.name, this will be semver-major. I'm OK with that.